.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+npm-debug.log
```

Comment:
Should we add npm-debug.log to the gitignore or just not have npm-debug logs at all? 😃